### PR TITLE
diatheke: add audio format to create session request

### PR DESCRIPTION
**Pull request by ppcssn**

### CODE REVIEW

File: proto/cobaltspeech/diatheke/v3/diatheke.proto [hunk 1]
```diff
@@ -133,6 +133,14 @@ message CreateSessionRequest {
   // session. The server may record this metadata when processing the
   // request. The server does not use this field for any other purpose.
   SessionMetadata metadata = 3;
+
+  // Format of the audio system expects to recieve. If no value specify, default
+  // value from the model will be use.
```
Comment (happyalu) [inline proto/cobaltspeech/diatheke/v3/diatheke.proto:138]:
```suggestion
  // Format of the audio system expects to recieve. This is an optional field, and if no value is specified, input will be assumed to be raw bytes (PCM16SLE) at the sample rate that speech processing models are configured to use on the server.
```

File: proto/cobaltspeech/diatheke/v3/diatheke.proto [hunk 2]
```diff
@@ -133,6 +133,14 @@ message CreateSessionRequest {
...
+  AudioFormat input_audio_format = 4;
+
+  // Format of the audio client expects to recieve. If no value specify, default
+  // value from TTS model will be use.
```
Comment (happyalu) [inline proto/cobaltspeech/diatheke/v3/diatheke.proto:142]:
```suggestion
  // Format of the audio client expects to recieve. This is an optional field. If no value is specified, the output will be produced as a WAV file with a sample rate that text-to-speech models are configured to use on the server. 
```

File: proto/cobaltspeech/diatheke/v3/diatheke.proto [hunk 3]
```diff
@@ -480,3 +488,80 @@ message SessionMetadata {
   // saved for this session.
   string storage_file_prefix = 2;
 }
+
+// Details of audio in format
+message AudioFormat {
```
Comment (happyalu) [inline proto/cobaltspeech/diatheke/v3/diatheke.proto:493]: Is it time to put this into a separate file somewhere? we use this quite often, no?

Comment (ppcssn) [reply proto/cobaltspeech/diatheke/v3/diatheke.proto:493]: it's only similar to what voicegen use.

**Review by happyalu — APPROVED** [review]

**Pull request merged**